src/components/views/elements/MemberEventListSummary.js

```diff
@@ -206,27 +207,27 @@ module.exports = React.createClass({
      * @returns {string} the written English equivalent of the transition.
      */
     _getDescriptionForTransition(t, plural, repeats) {
-        const beConjugated = plural ? "were" : "was";
-        const invitation = "their invitation" + (plural || (repeats > 1) ? "s" : "");
+        const beConjugated = plural ? _t('were') : _t('was');
+        const invitation = (plural || (repeats > 1) ? _t('their invitations') : _t('their invitation'));
 
         let res = null;
         const map = {
-            "joined": "joined",
-            "left": "left",
-            "joined_and_left": "joined and left",
-            "left_and_joined": "left and rejoined",
-            "invite_reject": "rejected " + invitation,
-            "invite_withdrawal": "had " + invitation + " withdrawn",
-            "invited": beConjugated + " invited",
-            "banned": beConjugated + " banned",
-            "unbanned": beConjugated + " unbanned",
-            "kicked": beConjugated + " kicked",
-            "changed_name": "changed name",
-            "changed_avatar": "changed avatar",
+            "joined": _t('joined'),
+            "left": _t('left'),
+            "joined_and_left": _t('joined and left'),
+            "left_and_joined": _t('left and rejoined'),
+            "invite_reject": _t('rejected') + ' ' + invitation,
+            "invite_withdrawal": _t('had') + ' ' + invitation + ' ' + _t('withdrawn'),
+            "invited": beConjugated + ' ' + _t('was invited'),
+            "banned": beConjugated + ' ' + _t('was banned'),
+            "unbanned": beConjugated + ' ' + _t('was unbanned'),
+            "kicked": beConjugated + ' ' + _t('was kicked'),
+            "changed_name": _t('changed name'),
+            "changed_avatar": _t('changed avatar'),
         };
 
         if (Object.keys(map).includes(t)) {
-            res = map[t] + (repeats > 1 ? " " + repeats + " times" : "" );
+            res = map[t] + (repeats > 1 ? ' ' + repeats + ' ' + _t('times') : '' );
```

Comment:
i guess this all assumes "noun, verb, adverb" word ordering as per English... i wonder how many languages this will cause a problem for...